test/e2e_test.go

```diff
@@ -110,34 +107,33 @@ var _ = Describe("e2e", func() {
 			Expect(block).ToNot(BeNil())
 
 			//create channel
-			var caFile, clientCert, clientKey, osnURL string
-			osnURL = "https://localhost:7053"
-			caFile = "../fabric-samples/test-network/organizations/ordererOrganizations/example.com/tlsca/tlsca.example.com-cert.pem"
-			clientCert = "../fabric-samples/test-network/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/tls/server.crt"
-			clientKey = "../fabric-samples/test-network/organizations/ordererOrganizations/example.com/orderers/orderer.example.com/tls/server.key"
-			caCertPool := x509.NewCertPool()
-			caFilePEM, err := os.ReadFile(caFile)
-			caCertPool.AppendCertsFromPEM(caFilePEM)
-			Expect(err).NotTo(HaveOccurred())
-			tlsClientCert, err := tls.LoadX509KeyPair(clientCert, clientKey)
-			Expect(err).NotTo(HaveOccurred())
-			resp, err := channel.CreateChannel(osnURL, block, caCertPool, tlsClientCert)
-			Expect(err).NotTo(HaveOccurred())
-			body, err := io.ReadAll(resp.Body)
-			if err != nil {
-				fmt.Println("my Http error is ", err)
-			}
-			fmt.Println("response statuscode is ", resp.StatusCode,
-				"\nhead[name]=", resp.Header["Name"],
-				"\nbody is ", string(body))
-			Expect(err).NotTo(HaveOccurred())
-			Expect(resp.StatusCode).Should(Equal(http.StatusCreated))
+			// var caFile, clientCert, clientKey, osnURL string
```

Comment:
by this PR, we will use gateway client to connect with fabric network.
Hence admin sdk does not need to connect to orderer node? just need to connect with peer?

Reply:
if so, I hope we can clean up unused code in comments.

Reply:
As much as practical, I would like to use the Gateway client. This means:

1. the end user only needs to establish a gRPC connection to a single endpoint address instead of making connections to multiple peers and orderers, and dealing with fail-over for them at the client end.
2. the admin SDK will work with no changes when BFT consensus is added in Fabric v3, which would not be the case otherwise.

I left the channel test code commented out as it really should be used, either in this test or in its own channel configuration test suite. Unfortunately it does not current produce a usable channel configuration since it does not configure any anchor peers and so service discovery does not work.